nemo/collections/nlp/modules/common/megatron/position_embedding/kerple_relative_position_embedding.py

```diff
@@ -65,21 +65,26 @@ def __init__(
 
         # initialize the slopes
         self.kerple_b = torch.nn.Parameter(build_slopes(num_attention_heads, num_attention_heads_kerple))
-        self.kerple_a = torch.zeros_like(self.kerple_b)
-        self.kerple_p = torch.ones_like(self.kerple_b)
+        self.kerple_a = torch.nn.Parameter(torch.rand_like(self.kerple_b))
```

Comment:
This initialization is unstable. Can you please change it to:
```
        self.kerple_a = torch.nn.Parameter(torch.ones_like(self.kerple_b))
        self.kerple_p = torch.nn.Parameter(torch.ones_like(self.kerple_b))
```

Reply:
Added